nucliadb/nucliadb/reader/app.py

```diff
@@ -81,25 +83,29 @@ async def client_disconnect_handler(request: Request, exc: ClientDisconnect):
 )
 
 
-base_app = FastAPI(title="NucliaDB Reader API", **fastapi_settings)  # type: ignore
+def get_application() -> FastAPI:
```

Comment:
I wonder is "create_application" would be a better word actually?

Reply:
yup. or application factory?